dts/arm/nxp/nxp_imx95_m7.dtsi

```diff
@@ -1,12 +1,13 @@
 /*
- * Copyright 2024 NXP
+ * Copyright 2025 NXP
  *
```

Comment:
Recommended practice here is to keep both years. Like this `Copyright 2024-2025 NXP`